clients/admin-ui/src/features/integrations/configure-monitor/ConfigureMonitorForm.tsx

```diff
@@ -70,6 +70,7 @@ const ConfigureMonitorForm = ({
       ? {
           ...monitor,
           ...executionInfo,
+          last_monitored: undefined,
```

Comment:
`last_monitored` is read-only and (doesn't currently but) will throw an error if it's sent to the backend.  See [here](https://ethyca.atlassian.net/browse/PROD-2145?focusedCommentId=41803) for context.